content/blog/2015-04-23-prometheus-posts-around-the-web.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Prometheus posts around the web"
+created_at: 2015-04-23
+kind: article
+author: Brian Brazil
+---
+
+It has been almost 3 months since Prometheus version [0.10.0 was publicly
+announced](https://developers.soundcloud.com/blog/prometheus-monitoring-at-soundcloud).
+
+SoundCloud's post remains the best overview of the key components of
+Prometheus, but there has been lots of other online activity around Prometheus.
+This post will let you catch up on anything you missed.
+
+## Using Prometheus
+
+Posts on how to use Prometheus compose the majority of online content. Here's the
```

Comment:
"compose" -> "comprise" may be better

"Here's" -> "Here are"

"part" -> "the part"